src/parsec_client/operations/psa_sign_message.md

```diff
@@ -0,0 +1,36 @@
+# PsaSignMessage
+
+Sign a message with a private key. Opcode: 23 (`0x0017`)
```

Comment:
`PsaMacVerify` and this one have the same opcode!

Reply:
👀